Documentation/dev-releases.md

```diff
@@ -86,18 +81,3 @@ sudo setenforce Permissive
 make docker-image
```

Comment:
We can get rid of the "rkt doesn't play nice with SELinux" comment under "Building the Docker image"